testing/test_sampler.py

```diff
@@ -76,13 +77,14 @@ def test_sampler(transforms_dict, train_lst, target_lst, roi_params):
                               collate_fn=imed_loader_utils.imed_collate,
                               num_workers=0)
     neg_percent, pos_percent = _cmpt_label(train_loader)
-    assert neg_percent > pos_percent
+    assert abs(neg_percent - pos_percent) > 20
```

Comment:
the images changes hence the change in the balance

Reply:
Not sure to understand why the previous test is not relevant anymore. COuld you please help me?

Reply:
The balance changed now And I wanted it to be robust to new images (in case we change them again)